components/user-mgt/org.wso2.carbon.identity.user.profile/src/main/java/org/wso2/carbon/identity/user/profile/mgt/association/federation/exception/FederatedAssociationManagerException.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License year

Reply:
Fixed in 78f08459d2c398277e992c0c738443cfe4edad1b